fastlane/Fastfile

```diff
@@ -140,11 +147,11 @@ lane :merge_main_to_develop do
 
   sh('git checkout main')
   sh('git pull origin main')
-  sh('git checkout develop')
+  sh('git checkout origin/develop')
   sh('git pull origin develop')
   sh('git log develop..main')
   sh('git merge main')
-  sh('git push')
+  sh('git push origin HEAD:develop')
```

Comment:
Tested this locally on SwiftUI and it worked fine. Still needs to be tested on CI on release.